oss-cloud-frontend/src/components/ContributionsPage.vue

```diff
@@ -17,13 +17,29 @@
       </template>
 
       <template v-slot:item.actions="{ item }">
-          <v-item-group row>
-          <v-btn fab dark color="green" height="30" width="30" class="mx-1" @click="updateStatus(item, 'Visible')"
-            v-if="item.status != 'Visible'">
+        <v-item-group row>
+          <v-btn

```

Comment:
Let's try to be consistent when defining button element (and other types too).
There are changes where every attribute is in separate column, but then there are others that put everything in one line -> [link](https://github.com/levinine/oss-cloud/pull/19/files#diff-1e0cbd97437ffcf8717aaa61f0a38acbR38) [link](https://github.com/levinine/oss-cloud/pull/19/files#diff-c16288988fe45317499ed65c9bf08c91R9)

I would suggest to put all attributes in one line, eventually events or flow attributes in separate lines.

Reply:
Tried three different formatters, all do the same.